google/cloud/spanner/samples/samples.cc

```diff
@@ -98,6 +100,20 @@ void AddColumn(std::vector<std::string> const& argv) {
   (argv[0], argv[1], argv[2]);
 }
 
+void QueryWithStruct(std::vector<std::string> const& argv) {
+  // TODO(tmatsuo): Add querying part once data client is ready.
+  // [START spanner_create_struct_with_data]
+  auto singer_info = std::make_tuple(std::make_pair("FirstName", "Elena"),
+                                     std::make_pair("LastName", "Campbell"));
+  // [END spanner_create_struct_with_data]
+  (void)argv;
```

Comment:
You can just omit the name `argv` from the function argument since we don't use the argument in this function. That *should* quiet whatever warning you were seeing.

Reply:
Done